metadataschemas.md

```diff
@@ -28,33 +28,53 @@ Provide detailed information about a specific metadata schema. The JSON response
 
 Exposed links: none
 
+Return codes:
+* **200 OK** - if the operation succeed
+* **404 Not Found** - if the metadata schema doesn't exist
+
 ## Creating a Metadata Schema
 
 **POST /api/core/metadataschemas**
 
 To create a metadata schema, perform a post with the JSON below when logged in as admin.
 
-```
+```json
 {
   "prefix": "example",
-  "namespace": "http://dublincore.org/documents/example/",
-  "type": "metadataschema"
+  "namespace": "http://dublincore.org/documents/example/"
 }
 ```
 
+Return codes:
+* **201 Created** - if the operation succeed
+* **400 Bad Request** - if the body of the request can't be parsed (for example because of a missing field)
+* **401 Unauthorized** - if you are not authenticated
+* **403 Forbidden** - if you are not logged in with sufficient permissions
+* **422 Unprocessable Entity** - if the request is well-formed, but is invalid based on the given data.
+  * If you attempt to create a schema with an empty `prefix` or if it contains dots.
```

Comment:
We could just say "invalid characters".  Otherwise, this should list them all...they currently appear to be "dots, commas, underscores or spaces."